lib/ansible/modules/system/firewalld.py

```diff
@@ -536,6 +674,11 @@ def main():
             interface=dict(required=False, default=None),
             masquerade=dict(required=False, default=None),
             offline=dict(type='bool', required=False, default=None),
+            chain=dict(required=False, default=None),
```

Comment:
We avoid adding `required=False` and `default=None`, but we must add `type=str`.
Same for the other options.
```suggestion
            chain=dict(type='str'),
```